### PR TITLE
test(e2e,playwright): show.tsx

### PR DESCRIPTION
**Pull request by anand-harsh**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

 #51705

image

I was getting above warning so I splitted the code into two seperate files.

### CODE REVIEW

File: e2e/show-button.spec.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import { test, expect, type Page } from '@playwright/test';
+import translations from '../client/i18n/locales/english/translations.json';
+
+let page: Page;
```
Comment (naomi-lgbt) [inline e2e/show-button.spec.ts:4]:
```suggestion
```

Don't need to declare this, because...

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

**Comment by anand-harsh** [comment]
Hi @naomi-lgbt , I have tried to resolve all issues. Kindly review and suggest if something more needed.
Without seed Title test is getting failed in chromium. So I added the seed only ion title and removed from other one. Thank you for helping with the issue.

**Comment by anand-harsh** [comment]
Hi @naomi-lgbt , I have tried to resolve all issues. Kindly review and suggest if something more needed.
Without seed,  Title test is getting failed in chromium. So I added the seed only in title and removed from other one. Thank you for helping with the issue.

**Comment by anand-harsh** [comment]
Hi @naomi-lgbt , I have resolved the issues. Kindly review and suggest if something more needed.
Without seed, Title test is getting failed in chromium. So I added the seed only in title and removed from other one. Thank you for helping with the issue.

**Comment by jdwilkin4** [comment]
HI @anand-harsh ! 

I removed the closes keyword from your PR because we don't want to close the hacktoberfest issue because it needs to stay open for multiple contributors 

You should only use the closes keyword when you want to automatically close an issue after the PR has been approved and merged.
In this case, we only want to simply mention it.

Thanks for understanding 👍

**Comment by anand-harsh** [comment]
> HI @anand-harsh !
> 
> I removed the closes keyword from your PR because we don't want to close the hacktoberfest issue because it needs to stay open for multiple contributors
> 
> You should only use the closes keyword when you want to automatically close an issue after the PR has been approved and merged. In this case, we only want to simply mention it.
> 
> Thanks for understanding 👍

Sure Sir

**Comment by anand-harsh** [comment]
Hi @naomi-lgbt, @huyenltnguyen , I have resolved the issues. Kindly review and suggest if something more needed.
Without seed, Title test is getting failed in chromium. So I added the seed only in title and removed from other one. Thank you for helping with the issue.

**Review by huyenltnguyen — COMMENTED** [review]
Looking at this again, I think we should consolidate the test files instead of having one for the button and one for the title.

**Comment by anand-harsh** [comment]
> Looking at this again, I think we should consolidate the test files instead of having one for the button and one for the title.

Done this change. I consolidated both files in a single file and it is running now.

**Comment by anand-harsh** [comment]
> Looking at this again, I think we should consolidate the test files instead of having one for the button and one for the title.

Hi @huyenltnguyen , Done this change. I consolidated both files in a single file and it is running now. Kindly check and suggest if more changes needed.

**Review by naomi-lgbt — APPROVED** [review]

**Comment by anand-harsh** [comment]
@huyenltnguyen I did the changes and deleted seed and beforAll.

**Review by naomi-lgbt — APPROVED** [review]
A quick note that you don't need to continuously merge `main` into this branch.

Also, you shouldn't work directly off of your `main` branch.

**Comment by anand-harsh** [comment]
> A quick note that you don't need to continuously merge `main` into this branch.
> 
> Also, you shouldn't work directly off of your `main` branch.

Sure, I will take care of this.

**Review by huyenltnguyen — APPROVED** [review]

**Pull request closed without merging**